drivers/hmis/spec/requests/hmis/assessments/submit_assessment_spec.rb

```diff
@@ -246,7 +246,7 @@
         },
         {
           'fullMessage' => 'The Required Field must exist',
-          'attribute' => 'fieldOne',
+          'attribute' => 'base',
```

Comment:
Huh, yeah it should be fine to change it, just adding `|| item.mapping&.custom_field_key` to the end of [this line](https://github.com/greenriver/hmis-warehouse/blob/a74e8d2f57d159655f6923d3b1dc354eb692742b/drivers/hmis/app/models/hmis/form/definition.rb#L439). It shouldn't make any difference since the errors are "wired up" based on the `link_id` that is resolved on each ValidationError, not the `attribute` (which is this field). Not ideal that we check for `attribute` here but not `link_id`, that should probably be added to the assertion.